<ui version="4.0" >
 <class>CExposureParamsGuiComp</class>
 <widget class="QWidget" name="CExposureParamsGuiComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="ShutterTimeLabel" >
     <property name="text" >
      <string>Shutter time</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QDoubleSpinBox" name="ShutterTimeSB" >
     <property name="maximum" >
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="ShutterTimeUnitLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>ms</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="DelayTimeLabel" >
     <property name="text" >
      <string>Delay Time</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QDoubleSpinBox" name="DelayTimeSB" >
     <property name="maximum" >
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="DelayTimeUnitLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>ms</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="EenDelayTimeLabel" >
     <property name="text" >
      <string>EEN Delay</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QDoubleSpinBox" name="EenDelayTimeSB" >
     <property name="maximum" >
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLabel" name="EenDelayTimeUnitLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>ms</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
